rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2363,6 +2363,40 @@ void voteFederationChange_commitEmptyFederation_returnsInsufficientMembersRespon
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsSuccessfulResponseCodeAndPendingFederationSizeZero() {
```

Comment:
nit, but i would say `_returnsSuccessfulResponseCode But PendingFederationSizeZero`

Reply:
No need. The test is checking `assertThat(federationSupport.getPendingFederationSize(), is(0));`, so this goes better with the name of the test.